### PR TITLE
Preview templates in parcel

### LINKED ISSUE #281: Preview each template in Parcel
[Here's the docs from Avi](https://parcel.io/docs/sharing/playground). I bet we could replace the "view template in a browser" button for each template with this. Seems like a reasonable upgrades.

### PR DESCRIPTION
**Pull request by TedGoas**
This PR replaces the "View in Browser" buttons with "Edit in Parcel" buttons.

Resolves #281

### CODE REVIEW

File: docs/content/templates.md [hunk 1]
```diff
@@ -69,3 +94,32 @@ I recommend against using a CSS inliner with Cerberus. Here’s why:
...
+
+<script>
+  (function () {
+    const forms = document.querySelectorAll('.parcel-button');
+    for (let i = 0; i < forms.length; i++) {
+      const form = forms[i];
+      form.addEventListener('click', function (event) {
+        const url = form.getAttribute('data-url');
+        // if we have a url to fetch html from
+        if (!form.elements.data.value && url) {
```
Comment (avigoldman) [inline docs/content/templates.md:106]: Totally silly of me – in the example I forgot to add the else condition:

Add this and then this looks great!
```
else {
form.submit()
}
```

**Review by avigoldman — APPROVED** [review]

File: docs/content/templates.md (hunk 1 above)
Comment (avigoldman) [reply docs/content/templates.md:106]: Without this, the second time you click on the button it doesn't work

Comment (TedGoas) [reply docs/content/templates.md:106]: @avigoldman [Like so?](https://github.com/TedGoas/Cerberus/pull/282/commits/1dab19f7e4957da393df95329372c2b429994b83)

Comment (avigoldman) [reply docs/content/templates.md:106]: Perfect!

**Review by avigoldman — APPROVED** [review]

**Pull request merged**